### PR TITLE
add: `kicad-diff-git`

### PR DESCRIPTION
**Pull request by MarcelRobitaille**
## Progress 

- [x] Edit packagelist
- [x] Add initial pacscript
- [ ] Contact devs
- [ ] Add maintainer to pacscript

### CODE REVIEW

File: packages/kicad-diff-git/kicad-diff-git.pacscript [hunk 1]
```diff
@@ -0,0 +1,31 @@
+name="kicad-diff-git"
+repology=("project: kicad-diff-git")
+version="4500fef3"
```
Comment (Henryws) [inline packages/kicad-diff-git/kicad-diff-git.pacscript:3]:
```suggestion
```

**Review by Henryws — CHANGES_REQUESTED** [review]

**Comment by MarcelRobitaille** [comment]
@Henryws thanks for reviewing. Is this what you meant?

File: packages/kicad-diff-git/kicad-diff-git.pacscript [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+build_depends=("git" "python3-pip")
+depends=("python3")
+gives="kicad-diff"
+description=" Scripts for performing image diffs between pcbnew layout revisions Resources"
+maintainer="Marcel Robitaille <mail@marcelrobitaille.me>"
+arch=("any")
+provides=("kicad-diff")
+conflicts=("kicad-diff")
+url="https://github.com/leoheck/KiCad-Diff.git"
+
```
Comment (Henryws) [inline packages/kicad-diff-git/kicad-diff-git.pacscript:14]:
```suggestion
```

**Review by Henryws — CHANGES_REQUESTED** [review]

File: packages/kicad-diff-git/kicad-diff-git.pacscript (hunk 2 above)
Comment (MarcelRobitaille) [reply packages/kicad-diff-git/kicad-diff-git.pacscript:14]: Isn't `url` required? It's used in `pkgver()` and to download the source code, right? If I try to remove it, I get:

```
(kicad-diff-git) Do you want to view/edit the pacscript [y/N]
[+] INFO: Sourcing pacscript
fatal: no path specified; see 'git help pull' for valid url syntax
fatal: no path specified; see 'git help pull' for valid url syntax
[!] ERROR: Package does not contain version
```

Comment (Henryws) [reply packages/kicad-diff-git/kicad-diff-git.pacscript:14]: I said to to remove the blank line, not URL.

Comment (MarcelRobitaille) [reply packages/kicad-diff-git/kicad-diff-git.pacscript:14]: Oh sorry the diff in GitHub makes it look like I should replace 11-14 with just an empty line

File: packages/kicad-diff-git/kicad-diff-git.pacscript [hunk 3]
```diff
@@ -0,0 +1,30 @@
+name="kicad-diff-git"
+repology=("project: kicad-diff-git")
+epoch="1"
```
Comment (Henryws) [inline packages/kicad-diff-git/kicad-diff-git.pacscript:3]: Why is epoch needed?

**Review by Henryws — CHANGES_REQUESTED** [review]

File: packages/kicad-diff-git/kicad-diff-git.pacscript [hunk 4]
```diff
@@ -0,0 +1,29 @@
+name="kicad-diff-git"
+repology=("project: kicad-diff-git")
+homepage="https://github.com/leoheck/KiCad-Diff"
+build_depends=("git" "python3-pip")
+depends=("python3")
+gives="kicad-diff"
+description=" Scripts for performing image diffs between pcbnew layout revisions Resources"
```
Comment (Henryws) [inline packages/kicad-diff-git/kicad-diff-git.pacscript:7]:
```suggestion
description="Scripts for performing image diffs between pcbnew layout revisions Resources"
```

File: packages/kicad-diff-git/kicad-diff-git.pacscript [hunk 5]
```diff
@@ -0,0 +1,29 @@
+name="kicad-diff-git"
+repology=("project: kicad-diff-git")
+homepage="https://github.com/leoheck/KiCad-Diff"
+build_depends=("git" "python3-pip")
```
Comment (Henryws) [inline packages/kicad-diff-git/kicad-diff-git.pacscript:4]:
```suggestion
build_depends=("python3-pip")
```

**Review by Henryws — CHANGES_REQUESTED** [review]

**Comment by Henryws** [comment]
What should we do about this?

**Comment by MarcelRobitaille** [comment]
@Henryws Sorry what should we do about what

**Comment by Henryws** [comment]
The CI failing.

**Comment by MarcelRobitaille** [comment]
Seems to be an error resulting from `pip install wxpython`:
```
ModuleNotFoundError: No module named 'attrdict'
```

The solution I've found online is to use Python 3.10. For things like this, I usually use pyenv.

**Comment by Henryws** [comment]
So why can't we use 3.10?

**Comment by MarcelRobitaille** [comment]
It probably has to do with the packaged version in Ubuntu. Maybe the pacstall script worked on my computer because I had pyenv, which injects its own version of python into $PATH. I'm traveling right now. Maybe when I'm back I could figure out a script with pyenv?

**Comment by Elsie19** [comment]
Stale.

**Pull request closed without merging**